packages/block-library/src/media-text/transforms.js

```diff
@@ -3,6 +3,25 @@
  */
 import { createBlock } from '@wordpress/blocks';
 
+function transform(
```

Comment:
Nit: maybe it’d be nice to name this `unwrap` or something that signifies what actually happens.